### PR TITLE
PrometheusAI group added

### PR DESCRIPTION
**Pull request by benlugasi**
Hey,
I've added PrometheusAI as we spoke, please review my changes and see if there is anything else that needs to be done.
Thanks,
Ben

### CODE REVIEW

File: cluster-scope/base/user.openshift.io/groups/PrometheusAI/group.yaml
Comment (4n4nd) [inline cluster-scope/base/user.openshift.io/groups/PrometheusAI/group.yaml:4]: @HumairAK should the name be all lowercase?

Comment (HumairAK) [reply cluster-scope/base/user.openshift.io/groups/PrometheusAI/group.yaml:4]: yep, [ref](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)

**Comment by benlugasi** [comment]
Hey,
I've changed everything to lowercase waiting for your approval

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Comment by HumairAK** [comment]
@benlugasi can you squash your commits down to 1 commit

**Comment by benlugasi** [comment]
/retest
I squashed the commits, and fetched the 'apps' master so you'll see other changes in it

**Comment by 4n4nd** [comment]
@benlugasi if you look at the [changes](https://github.com/operate-first/apps/pull/1304/files), you can see there are changes from other commits that you did not make. Once you have fetched from the upstream apps/master, you need to do a rebase. This will add the new changes to your branch.

**Pull request closed without merging**
